app/src/main/res/values/strings.xml

```diff
@@ -55,5 +55,10 @@
     <string name="get_card_button">Get Card</string>
     <string name="set_card_button">Add Card</string>
     <string name="manage_collection_button">Manage Collection</string>
+    <string name="return_to_home">Go back</string>
+    <string name="export_my_collection">Export my collection</string>
```

Comment:
I recommend you to create a suffix for all strings related to your fragment
Example : ExportCollection_title
When you have lots of strings, it can be better to distinguish them.